Client/Frontend/Browser/BrowserToolbar.swift

```diff
@@ -78,42 +80,49 @@ public class BrowserToolbarHelper: NSObject {
+        toolbar.stopReloadButton.setImage(UIImage.templateImageNamed("bottomNav-refresh"), forState: .Normal)
+        toolbar.stopReloadButton.setImage(UIImage(named: "bottomNav-refreshEngaged"), forState: .Highlighted)
         toolbar.stopReloadButton.accessibilityLabel = NSLocalizedString("Reload", comment: "Accessibility Label for the browser toolbar Reload button")
         let longPressGestureStopReloadButton = UILongPressGestureRecognizer(target: self, action: #selector(BrowserToolbarHelper.SELdidLongPressStopReload(_:)))
         toolbar.stopReloadButton.addGestureRecognizer(longPressGestureStopReloadButton)
         toolbar.stopReloadButton.addTarget(self, action: #selector(BrowserToolbarHelper.SELdidClickStopReload), forControlEvents: UIControlEvents.TouchUpInside)
 
-        toolbar.shareButton.setImage(UIImage(named: "send"), forState: .Normal)
-        toolbar.shareButton.setImage(UIImage(named: "sendPressed"), forState: .Highlighted)
+        toolbar.shareButton.setImage(UIImage.templateImageNamed("bottomNav-send"), forState: .Normal)
+        toolbar.shareButton.setImage(UIImage(named: "bottomNav-sendEngaged"), forState: .Highlighted)
         toolbar.shareButton.accessibilityLabel = NSLocalizedString("Share", comment: "Accessibility Label for the browser toolbar Share button")
         toolbar.shareButton.addTarget(self, action: #selector(BrowserToolbarHelper.SELdidClickShare), forControlEvents: UIControlEvents.TouchUpInside)
-        toolbar.bookmarkButton.contentMode = UIViewContentMode.Center
 
-        toolbar.bookmarkButton.setImage(UIImage(named: "bookmark"), forState: .Normal)
-        toolbar.bookmarkButton.setImage(UIImage(named: "bookmarked"), forState: UIControlState.Selected)
-        toolbar.bookmarkButton.setImage(UIImage(named: "bookmarkHighlighted"), forState: UIControlState.Highlighted)
-        toolbar.bookmarkButton.accessibilityLabel = NSLocalizedString("Bookmark", comment: "Accessibility Label for the browser toolbar Bookmark button")
-        let longPressGestureBookmarkButton = UILongPressGestureRecognizer(target: self, action: #selector(BrowserToolbarHelper.SELdidLongPressBookmark(_:)))
-        toolbar.bookmarkButton.addGestureRecognizer(longPressGestureBookmarkButton)
-        toolbar.bookmarkButton.addTarget(self, action: #selector(BrowserToolbarHelper.SELdidClickBookmark), forControlEvents: UIControlEvents.TouchUpInside)
+        if AppConstants.MOZ_MENU {
+            toolbar.menuButton.contentMode = UIViewContentMode.Center
+            toolbar.menuButton.setImage(UIImage.templateImageNamed("bottomNav-menu"), forState: .Normal)
+            toolbar.menuButton.accessibilityLabel = NSLocalizedString("Menu", comment: "Accessibility Label for the browser toolbar Menu button")
+            toolbar.menuButton.addTarget(self, action: #selector(BrowserToolbarHelper.SELdidClickMenu), forControlEvents: UIControlEvents.TouchUpInside)
```

Comment:
Nit: With the new #selector form, would it make sense to start dropping the SEL prefix?